src/applications/check-in/api/versions/v2.js

```diff
@@ -6,7 +6,7 @@ import environment from '@department-of-veterans-affairs/platform-utilities/envi
 import { makeApiCallWithSentry } from '../utils';
 
 const v2 = {
-  getSession: async ({ token, checkInType }) => {
+  getSession: async ({ token, checkInType = 'day-of' }) => {
```

Comment:
Why this change? If you change this here, we would need changes down in the code too. But I think this was purposely built this way to not add the query string for day-of.

Reply:
I meant to remove this as well thanks for catching it.